pkg/scheduler/plugins/elasticquota/core/group_quota_manager.go

```diff
@@ -555,6 +555,19 @@ func (gqm *GroupQuotaManager) GetPodIsAssigned(quotaName string, pod *v1.Pod) bo
 	return quotaInfo.GetPodIsAssigned(pod)
 }
 
+func (gqm *GroupQuotaManager) MigratePodFromLeftToRight(pod *v1.Pod, lhs, rhs string) {
```

Comment:
MigratePodFromLeftToRight-> MigratePodBetweenQuotas; lhs->out, rhs->in ?

Reply:
fixed